src/Components/Home/NewQuestionModal.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Modal, Form, Button } from "react-bootstrap";
+
+const NewQuestionModal = (props) => {
+  return (
+    <Modal show={props.show} onHide={props.onHide}>
+      <Modal.Header>
+        <Modal.Title>New Question</Modal.Title>
```

Comment:
I'm not sure about the title. It attracts attention but doesn't provide any new useful information.

Reply:
I changed in the last commit.

Reply:
Now it allows the user to increase the product score even when there are no more questions to answer.